### PR TITLE
Accessibility: Communicate page state on screen reader

### PR DESCRIPTION
**Pull request by annielovesjs**
Before: 
- when you go to any of the links in the top nav bar, it just reads go to {page} page but does not communicate the state (ex. are we currently on {page}, or is {page} disabled?)

Changes:
- using NVDA screen reader, when the screen reader goes over the currently selected page, it should read current on {page}
- when the screen reader goes over a disabled page, it should read {page} disabled
- if the page is visited and not disabled or selected, it should just read {page} page

### CODE REVIEW

File: src/client/src/components/TopNavBarLink/index.tsx [hunk 1]
```diff
@@ -39,20 +39,34 @@ const TopNavBarLink = ({
       onClick={handleClick}
       className={styles.container}
       onKeyUp={keyUpHandler}
+      aria-label={
+        visitedCheck || isSelected
```
Comment (jcoc611-microsoft) [inline src/client/src/components/TopNavBarLink/index.tsx:59]: should we have `(..)`  parenthesis here?

Comment (SahilTara) [reply src/client/src/components/TopNavBarLink/index.tsx:59]: Honestly I would extract some of this out into a const function or two. It is kinda hard to read/follow.

Comment (streetcoder123) [reply src/client/src/components/TopNavBarLink/index.tsx:59]: @annielovesjs

Comment (annielovesjs) [reply src/client/src/components/TopNavBarLink/index.tsx:59]: @jcoc611-microsoft the parenthesis gets automatically removed

Comment (annielovesjs) [reply src/client/src/components/TopNavBarLink/index.tsx:59]: @SahilTara good point! I just abstracted it to a function call now so it should be more readable

**Review by SahilTara — APPROVED** [review]

**Review by jcoc611-microsoft — APPROVED** [review]

**Review by jsondoo — APPROVED** [review]
great work, tested with NVDA on windows

**Pull request merged**